### ISSUE TITLE
VSCode treating javascript code almost as plain text

### ISSUE THREAD

**Issue by nmaxcom** [Microsoft/vscode#69265]
Issue Type: <b>Bug</b>

I've been having this issue lately. I will get curious about source online, and I copy&paste it on VSCode to have a better and improved look. But it's just not happening.

* Syntax highlighting? Yes.
* Reformat with Prettier works some times
* Highlight matches of the word with the caret? Fewer times (and I think only if I temporarily save the file on disk)
* Any form of parsing or static analysis? Nope

As an example. I pasted a code about 30k LOC human formatted with no issues. Select Language Mode: JS (which it is). Then I want to go to the definition of a function, so I CMD+Click: nothing. With right click nothing again:
![image](https://user-images.githubusercontent.com/8936090/53284023-86335a80-374e-11e9-8bea-2363cb8a23b2.png)

I manually search for it and.. ta-dá:
![image](https://user-images.githubusercontent.com/8936090/53284038-ae22be00-374e-11e9-9f7b-cd0dd8073dc2.png)

I don't know what is going on. How to fix this?

VS Code version: Code 1.31.0 (7c66f58312b48ed8ca4e387ebd9ffe9605332caa, 2019-02-05T22:32:14.164Z)
OS version: Darwin x64 16.7.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz (8 x 2900)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (0.12GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (53)</summary>

Extension|Author (truncated)|Version
---|---|---
paradox-syntax|ada|1.0.1
rtf|ale|2.0.0
vscode-color|ans|0.4.5
vscode-intelephense-client|bme|1.0.2
npm-intellisense|chr|1.3.0
path-intellisense|chr|1.4.2
jsrefactor|cms|2.15.1
bracket-pair-colorizer|Coe|1.0.61
transformer|dak|1.6.0
vscode-eslint|dba|1.8.0
vscode-html2pug|dba|0.1.0
vscode-html-css|ecm|0.2.0
vscode-npm-script|eg2|0.3.5
prettier-vscode|esb|1.8.1
php-intellisense|fel|2.3.10
auto-close-tag|for|0.5.6
code-runner|for|0.9.7
html-slim-scss-css-class-completion|gen|1.5.12
pug2html|gin|0.0.5
rest-client|hum|0.21.1
symbolstreeview|igb|0.1.5
eml|lei|0.3.2
expand-region|let|0.1.2
render-crlf|med|1.2.1
dotenv|mik|1.0.1
vscode-jade-snippets|mrm|1.0.1
vscode-pugbeautify|mrm|1.0.2
vscode-puglint|mrm|2.3.0
mssql|ms-|1.4.0
python|ms-|2019.1.0
node-modules-resolve|nau|1.0.2
leaper|Onl|0.5.3
vscode-code-outline|pat|0.2.1
material-icon-theme|PKi|3.6.3
vscode-css-peek|pra|2.2.0
bracketeer|pus|1.1.0
LiveServer|rit|5.5.1
sass-indented|rob|1.5.1
bash-debug|rog|0.3.3
bash-beautify|sha|0.1.1
file-name-search|Sha|0.0.4
vscode-zipexplorer|sle|0.3.1
cwtools-vscode|tbo|0.7.3
paradox-syntax|tbo|0.1.7
es-quotes|vil|0.2.6
colonize|vms|2.2.2
quokka-vscode|Wal|1.0.189
vscode-todo-highlight|way|1.0.4
change-case|wma|1.0.0
vscode-jumpy|wma|0.3.1
vscode-surround|yat|1.0.2
file-size|zh9|0.1.3
propertylist|zho|0.0.2

(2 theme extensions excluded)

</details>

**Comment by mjbvz** [comment]
Please share some example code that demonstrates this problem

**Comment by mjbvz** [comment]
Closing as we need a complete code example (as text) to investigate this

**Comment by nmaxcom** [comment]
I don't remember where I got that file. I thought of recording a video with jQuery 3.3.1 from the cdn and then [another js file (90k LOC) from Cloud9](https://static.c9.io/nc-3.1.4355-0d1121ad-wf/config/profile.js).

I uploaded it to Youtube, but it's unlisted:
https://www.youtube.com/watch?v=OstVqRnzHKg

It used to go flawlessly doing things like that, but I don't know when or why it changed.

VSCode: 1.32.1 (1.32.1)
macOS Sierra 10.12.6
